### ISSUE TITLE
RFE: Add ability to filter export by using filters

### ISSUE THREAD

**Issue by MagicFab** [nuxsmin/sysPass#774]
I'd like to suggest the ability to filter the export result based on Favorites, Categories and/or Customer account (like the search form).

The only entries tagged as favorites, matching the category and or Customer account would be exported.

**Comment by nuxsmin** [comment]
Hello, this feature will be implemented in next release, by exporting the current accounts search filtering results.

Thanks for the feedback!

**Comment by MagicFab** [comment]
Is "next release" 3.0?

**Comment by nuxsmin** [comment]
Right, new features will be implemented since beta release. I've done a lot of changes to base code that avoid to implement some features until they were ready.

Regards.

**Comment by nuxsmin** [comment]
@MagicFab did you receive an email regarding the donation?

**Comment by MagicFab** [comment]
@nuxsmin yes, tx.!

**Comment by nuxsmin** [comment]
@MagicFab great!.. I had some issues sending some mails in the past. Thanks for the ack!
